plugins/qlcplus_4.12.0/export.js

```diff
@@ -759,16 +759,25 @@ function addHeads(xmlMode, mode) {
  * @returns {string} The first of the fixture's categories that is supported by QLC+, defaults to 'Other'.
  */
 function getFixtureType(fixture) {
-  const ignoredCats = [`Blinder`, `Matrix`, `Pixel Bar`, `Stand`];
+  // see https://github.com/OpenLightingProject/open-fixture-library/issues/581
+  if (fixture.categories.includes(`Pixel Bar`)) {
+    return isBeamBar() ? `LEDBarBeams` : `LEDBarPixels`;
```

Comment:
It should actually be `LED Bar (Beams)` and `LED Bar (Pixels)` (see mcallegari/qlcplus@edcbbfed9439108afa84dec754ed7bf3c81e9c04). The names without spaces are just QLC+' internal class names.

```suggestion
    return isBeamBar() ? `LED Bar (Beams)` : `LED Bar (Pixels)`;
```